packages/test/mocha-test-setup/mocharc-common.cjs

```diff
@@ -41,6 +41,14 @@ function getFluidTestMochaConfig(packageDir, additionalRequiredModules, testRepo
 		return mod;
 	});
 
+	if (process.env.FLUID_TEST_LOGGER_PKG_SPECIFIER) {
+		const modulePath = path.join(moduleDir, process.env.FLUID_TEST_LOGGER_PKG_SPECIFIER);
+		// Inject implementation of createTestLogger, put it first before mocha-test-setup
```

Comment:
I _think_ we don't need to put the new on in `requiredModulePaths` because we're going to `import` it directly in `mochaHooks.ts`. It matters in the current approach because we never import/require that one, we just expect it to have been loaded so its global definitions exist. But since we'll actually `import()` now, all the changes to this file might be unnecessary. Let's try that locally (reverting the changes to this file), I suspect things will work as expected.